src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/PackageAnnotationCheck.java

```diff
@@ -48,6 +48,20 @@
  * <pre>
  * &lt;module name=&quot;PackageAnnotation&quot;/&gt;
  * </pre>
+ * <p>Example of validating MyClass.java</p>
```

Comment:
End line with semicolon to show the separation.
`Example of validating MyClass.java:`

Do this for the others too.

Reply:
Done